tools/protoxform/protoxform.bzl

```diff
@@ -15,4 +15,33 @@ def _protoxform_impl(target, ctx):
 #
 #   bazel build //api --aspects tools/protoxform/protoxform.bzl%protoxform_aspect \
 #       --output_groups=proto
+#
 protoxform_aspect = api_proto_plugin_aspect("//tools/protoxform", _protoxform_impl, use_type_db = True)
+
+def _protoxform_rule_impl(ctx):
+    deps = []
+    for dep in ctx.attr.deps:
+        for path in dep[OutputGroupInfo].proto.to_list():
+            envoy_api = (
+                path.short_path.startswith("../envoy_api") or
+                path.short_path.startswith("../com_github_cncf_udpa")
```

Comment:
not entirely sure if this is either necessary or correct, but doesnt seem to affect outcome